bin/stellar-core-backup

```diff
@@ -17,20 +18,25 @@ OptionParser.new do |opts|
     options[:list] = l
   end
   opts.on("-r", "--restore", "offline restore, uses latest backup") do |r|
-    options[:restore] = r
+    options[:type] = 'restore'
+  end
+  opts.on("-R", "--rm", "auto-clean buckets directory") do |r|
+    options[:clean] = 'R'
+  end
+  opts.on("--no-verify", "Don't PGP sign backed up shasum file or verify restore files integrity (default on)") do |r|
+    options.delete(:verify)
+  end
+  opts.on("-d", "--debug", "dump debug output") do |r|
+    $debug = 'on'
   end
 end.parse!
 
-if options[:backup] then
-  scb = StellarCoreBackup::Job.new(
-    :config => options[:config],
-    :type   => 'backup'
-  )
-  scb.run()
-elsif options[:restore]
-  scb = StellarCoreBackup::Job.new(
-    :config => options[:config],
-    :type   => 'restore'
-  )
+if ! options[:type]
```

Comment:
To keep the code homogeneous, I think it would be nicer to use the `then` statement separator instead of `\n`.

Reply:
I am happy to comply with this. We should capture this as a style guide policy.

Reply:
I am not sure a style guide is needed for this, but definitely a good idea to keep the code the same. If you feel it's best to remove the `then`'s then we should do so everywhere.